pkg/config/config.go

```diff
@@ -10,6 +10,7 @@ type Install struct {
 	Device             string `json:"device,omitempty"`
 	ConfigURL          string `json:"configUrl,omitempty"`
 	ISOURL             string `json:"isoUrl,omitempty"`
+	DockerImage        string `json:"dockerImage,omitempty"`
```

Comment:
Since we are here, I would go for `containerImage` instead to be exposed to the user. We are not really bound to docker afterall (same should be propagated to the other code areas imho)

Reply:
well, I set it like this to match the elemental stuff, but indeed containerImage sounds better...maybe we should change it on elemental..

Reply:
The PR title should be adapted as well ;-)